flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/MesosTaskManagerParameters.java

```diff
@@ -238,6 +254,12 @@ public static MesosTaskManagerParameters create(Configuration flinkConfig) {
 			cpus = Math.max(containeredParameters.numSlots(), 1.0);
 		}
 
+		double gpus = Math.floor(flinkConfig.getDouble(MESOS_RM_TASKS_GPUS, 0.0));
```

Comment:
Are we rounding down because GPUs, cannot be shared? If this is the case, why don't we restrict the Flink configuration value to be an integer?

Reply:
My feedback was similar, we could simply parse as an integer. The value must be an integer due to a limitation in Mesos.  But the present solution seemed OK.

Reply:
@tillrohrmann I took possible changes in the future into consideration: what if Mesos accepts a float for # GPUs? If both of you @tillrohrmann and @EronWright think that taking an integer seems better, I'm going to follow your opinion 😃

Reply:
From the user perspective I think it's clearer to make it an integer because then we don't confuse users who haven't read the code and think that they can configure a fraction of a GPU based on its type. Once Mesos accepts floats, we can change it in Flink as well. I will apply the change while merging the PR.